src/k8s/templates/fedora28.template.js

```diff
@@ -0,0 +1,98 @@
+export const fedora28 = {
```

Comment:
please rename folte `templates` to `mock_templates` to avoid confusion.

These templates are expected to be loaded from API once implemented within kubevirt Operator (or so)

Reply:
renamed to `mock_templates`